app/src/main/java/io/devsummit/devsummit/Activities/LoginActivity.java

```diff
@@ -81,12 +80,12 @@
     private EditText mEmailView;
     private EditText mPasswordView;
     private View mProgressView;
-    private View mLoginFormView;
     private Button mLoginButton;
     private Button mLoginPhoneButton;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+        authHelper = new User_Authentication_Helper(this);
```

Comment:
why are you using this kind of case? Java usually just use CamelCase without any underscore. Except it is property. @caudaganesh

Reply:
okay i will fix it sir